types/defines/d1.d.ts

```diff
@@ -1,7 +1,14 @@
+interface D1Meta {
+  duration: number;
+  size_after: number;
+  rows_read: number;
+  rows_written: number;
+}
```

Comment:
```suggestion
  last_row_id: number;
  changed_db: boolean;
  changes: number;
}
```
Ah may as well type these too

Reply:
Thanks, I've added these in the latest commit